src/style.css

```diff
@@ -1,3 +1,62 @@
-body {
+.body {
```

Comment:
No need to add `body` class. You can use `body` tag in CSS, as well as `html`. The other tags should not be used in selectors.